scripts/variantstore/variant_annotations_table/README.md

```diff
@@ -0,0 +1,111 @@
+# Creating the Variant Annotations Table
+
+### The VAT pipeline is a set of WDLs
+- [GvsCreateVAT.wdl](scripts/variantstore/wdl/GvsCreateVAT.wdl)
+- [GvsValidateVAT.wdl](scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl)
+
+The pipeline takes in a jointVCF and outputs a table in BigQuery.
+
+**GvsCreateVAT** creates the table, while
+**GvsValidateVAT** checks and validates the VAT
+
+
+### Run GvsCreateVAT:
+
+Most of the inputs are constants — like the reference, or a table schema — and dont require any additional work (and paths can be found in the [example inputs json](scripts/variantstore/wdl/GvsCreateVAT.example.inputs.json)). However for the specific data being put in the VAT, three inputs need to be created.
+The first two of these inputs are two files — one of the file/vcf/shards you want to use for the VAT, and their corresponding index files. These are labelled as `inputFileofFileNames` and `inputFileofIndexFileNames`.
+
+You will need to know where all of the GVS extract output files live.
+Example of creating the files of file names and indices:
+`gsutil ls gs://prod-drc-broad/beta-release-99k-v3/beta_99k_*.vcf.gz > inputFileofFileNames.txt`
+`gsutil ls gs://prod-drc-broad/beta-release-99k-v3/beta_99k_*.vcf.gz.tbi > inputFileofIndexFileNames.txt`
+
+Those two created files will then need to get copied into a GCP bucket that this pipeline will have access to (eg. this bucket: `gs://aou-genomics-curation-prod-processing/vat/`)
+
+The third input is the ancestry file that Lee has made and it needs to be copied into a GCP bucket that this pipeline will have access to (eg. this bucket: `gs://aou-genomics-curation-prod-processing/vat/`) for easy access during the workflow. This input has been labelled as the `ancestry_file`.
```

Comment:
Not sure how much clarity this adds, since it doesn't explain what data is in the ancestry file.